src/main/java/org/opentripplanner/standalone/config/framework/project/EnvironmentVariableReplacer.java

```diff
@@ -63,14 +63,22 @@ public class EnvironmentVariableReplacer {
    *                                  environment variable do not exist.
    */
   public static String insertEnvironmentVariables(String text, String source) {
+    return insertEnvironmentVariables(text, source, System.getenv());
+  }
+
+  public static String insertEnvironmentVariables(
+    String text,
+    String source,
+    Map<String, String> variables
```

Comment:
What about `Function<String, String> getEnvironmentVariable` That way we could just fill all envvars with an empty string in the test, and not need to keep track of those?

Reply:
That's a good idea, which I just implemented.